libmscore/tremolo.cpp

```diff
@@ -229,31 +241,30 @@ void Tremolo::computeShape()
 //   layoutOneNoteTremolo
 //---------------------------------------------------------
 
-void Tremolo::layoutOneNoteTremolo(qreal x, qreal y, qreal _spatium)
+void Tremolo::layoutOneNoteTremolo(qreal x, qreal y, qreal spatium)
       {
       Q_ASSERT(!twoNotes());
 
       bool up = chord()->up();
       int line = up ? chord()->upLine() : chord()->downLine();
 
       if (!placeMidStem()) {
-            const qreal td = score()->styleS(Sid::tremoloDistance).val();
-            const qreal sw = score()->styleS(Sid::tremoloStrokeWidth).val();
-
             qreal t = 0.0;
             // nearest distance between note and tremolo stroke should be no less than 3.0
             if (chord()->hook() || chord()->beam()) {
-                  t = up ? -3.0 - (2.0 * (lines() - 1)) * td - 2.0 * sw : 3.0;
+                  t = up ? -3.0 - 2.0 * minHeight() : 3.0;
                   }
-            else { 
+            else {
+                  const qreal offset = 2.0 * score()->styleD(Sid::tremoloStrokeWidth);
```

Comment:
`Sid::tremoloStrokeWidth` is a Spatium value, and you are trying to read it as a double. This is causing a few of the mtests to crash. You should write this as
```
const qreal offset = 2.0 * score()->styleS(Sid::tremoloStrokeWidth).val();
```

Reply:
Oh thanks! That really clears things up.